03/app.js

```diff
@@ -42,11 +42,30 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    if(typeof this[isbn] === 'undefined') {
+        return null;
+    }
+
+    const book = this[isbn]['title'][lang];
+    
+    if(book) {
+        return book;
+    }
 
+    return false;
 }
 
 books.getTranslator = function(isbn, lang) {
 
+    if(this[isbn] === null) {
```

Comment:
Warto jeszcze sprawdzić czy nie jest równy `undefined` tak jak w linii 45 - `null`, a `undefined` to dwie różne wartości.

Reply:
Warunek został dodany.